Dockerfile.assisted-test-infra

```diff
@@ -1,6 +1,6 @@
 FROM quay.io/edge-infrastructure/assisted-service:latest AS service
 
-FROM quay.io/centos/centos:stream9
+FROM quay.io/assisted-installer-ops/base-python:3.11.0
```

Comment:
should we move the dockerfile behind this image publicly in github ?

Reply:
You mean moving only the dockerfile, or moving the entire repo from gitlab to github and also using some other ci/cd env?

Reply:
yeah I meant with the ci/cd, that's sad because we spend so much time to build it with gitlab